test/jdk/java/nio/channels/DatagramChannel/NotBound.java

```diff
@@ -24,19 +24,33 @@
 /* @test
  * @bug 4512723 6621689
  * @summary Test that connect/send/receive with unbound DatagramChannel causes
- *     the channel's socket to be bound to a local address
+ *     the channel's socket to be bound to a local address. This test may fail
+ *     intermittently on macOS if other datagram channel tests are running
+ *     concurrently on the same host.
```

Comment:
I don't think we should add this comment. Instead, if there is interference possible on the same host then we should create a TEST.properties in this directory with `exclusiveAccess.dirs=.` so that the tests don't run concurrently.

Reply:
OK. Note that interference can also come from other tests in other directories (like DatagramSocket tests in jdk/java/net/..) or from system processes (like syslogd on macOS which binds to an ephemeral port).